pkg/network/ebpf/c/tags-types.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __TAGS_TYPES_H
+#define __TAGS_TYPES_H
+
+#include "tracer.h"
+
+// dynamic tags
+#define TAGS_MAX_LENGTH 16
```

Comment:
16 bytes is really limiting. Let's try the perf buffer approach we talked about.